app/src/main/java/com/example/student_app_assignment/NewStudentActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.student_app_assignment
+
+import android.os.Bundle
+import android.util.Log
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import com.example.studentsapp.Student
+
+class NewStudentActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_add_student)
+
+        findViewById<Button>(R.id.addButton).setOnClickListener {
+            val id = findViewById<EditText>(R.id.idInput).text.toString()
+            val name = findViewById<EditText>(R.id.nameInput).text.toString()
+            val phone = findViewById<EditText>(R.id.phoneInput).text.toString()
```

Comment:
make all of this a private fun that gets the input and return a string since it's duplicated